common/makefile_common.mk

```diff
@@ -159,18 +153,17 @@ static_exe_robin_desktop: BOOST_LIB_IO=$(HTSSRC)/boost/lib/libboost_iostreams.a
 static_exe_robin_desktop: BOOST_LIB_PO=$(HTSSRC)/boost/lib/libboost_program_options.a
 static_exe_robin_desktop: $(EXEFILE)
 
-
 #COMPILATION RULES
 all: desktop
 
 $(BFILE): $(OFILE)
-	$(CXX) $(LDFLAG) $^ -o $@ $(DYN_LIBS)
+	$(CXX) $(LDFLAGS) $^ -o $@ $(DYN_LIBS)
```

Comment:
For using non-standard htslib and boost paths, I needed to modify this to successfully compile and run the binaries, where `HTSSRC_LIB=/opt/htslib/lib` and `BOOST_LIB=/opt/boost/lib`:
```
$(BFILE): $(OFILE)
	$(CXX) $(LDFLAGS) $^ -o $@ $(DYN_LIBS) -Wl,-rpath=$(BOOST_LIB) -Wl,-rpath=$(HTSSRC_LIB)
```
I also needed to add `LDFLAGS+= -L$(HTSSRC_LIB) -L$(BOOST_LIB)`